### PR TITLE
Update comment for method

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: src/c_array.c [hunk 1]
```diff
@@ -26,7 +26,7 @@ static void c_array_size(mrb_vm *vm, mrb_value *v)
 }
 
 
-// Array - size
+// Array - []]
```
Comment (murasesyuka) [inline src/c_array.c]: typo?
s/ []] / [] /g

Comment (badboy) [inline src/c_array.c]: D'oh!

**Comment by kaz0505** [comment]
Thank you!
Updated.

File: src/c_array.c (hunk 1 above)
Comment (kaz0505) [inline src/c_array.c]: Thank you.
Updated.

**Pull request merged**